### DISCUSSION TITLE
delete all files

### DISCUSSION THREAD

**Discussion by andrewbanchich** [ente-io/ente#3065] [Enhancements]
I've been trying Ente out to see if I want to migrate, but I can't find an easy way to delete everything. I'd like to start from scratch, so it would be really nice if there were a "Delete All" feature.

**Comment by ashilkn** [comment]
On the mobile app, to delete all:
1. Select an item from the home gallery.
2. Click on 'All" button, which will select all items in the home gallery.
3. Click on delete.
